### DISCUSSION TITLE
Inline icons in input fields

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#5802] [Help]
Is it possible to use inline icons in text/date inputs:

![image](https://user-images.githubusercontent.com/7851696/221578087-be42a65f-0340-4107-b005-b756cf338eed.png)

![image](https://user-images.githubusercontent.com/7851696/221578113-c00106df-f2c5-4a25-85e9-29c176dfb32f.png)

**Comment by Z3d0X** [comment]
Some form components, such as `TextInput`/`Select` have a `->prefixIcon()` method

```php
Select::make('consignee')
    ->prefixIcon('heroicon-o-office-building')
    ...
```

**Reply by oliverbj** [reply]
Thank you, @Z3d0X  - but this creates an icon _outside_ the input field. 
Wondering if there is an option to create the icon inside the input field.

**Reply by danharrin** [reply]
Not in v2, in v3 the icons are inside the input
